Update README.rst (#810)
python-control/python-control: README.rst

 :target: https://mybinder.org/v2/gh/python-control/python-control/HEAD

Features
--------

- Linear input/output systems in state-space and frequency domain
- Block diagram algebra: serial, parallel, feedback, and other interconnections
- Time response: initial, step, impulse
- Frequency response: Bode, Nyquist, and Nichols plots
- Control analysis: stability, reachability, observability, stability margins, root locus

> - Control design: eigenvalue placement, linear quadratic regulator, sisotool, hinfsyn

Author: ```suggestion
- Control design: eigenvalue placement, linear quadratic regulator, sisotool, hinfsyn, rootlocus_pid_designer
```

toplevel `python-control` directory::

  pip install .
  
Article and Citation Information
================================

> A `2021 article <https://ieeexplore.ieee.org/abstract/document/9683368>`_ about the library is available on IEEE Explore. If the Python Control Systems Library helped you in your research, please cite::

Author: ```suggestion
An `article <https://ieeexplore.ieee.org/abstract/document/9683368>`_ about the library is available on IEEE Explore. If the Python Control Systems Library helped you in your research, please cite::
```

  @inproceedings{python-control2021,

>     title={The python control systems library (python-control)},

Reviewer: I think we should capitalize Python or capitalize the entire title (which is how it appears on [IEEE Explore](https://ieeexplore.ieee.org/document/9683368): "The Python Control Systems Library (python-control)".

>     title={The python control systems library (python-control)},

Author: ```suggestion
    title={The Python Control Systems Library (python-control)},
```

    author={Fuller, Sawyer and Greiner, Ben and Moore, Jason and Murray, Richard and van Paassen, Ren{\'e} and Yorke, Rory},

>     booktitle={2021 60th IEEE Conference on Decision and Control (CDC)},

Reviewer: Remove 2021 here, since it appears below.

Author: Google Scholar's bibtex downloads are often a little off

>     booktitle={2021 60th IEEE Conference on Decision and Control (CDC)},

Author: ```suggestion
    booktitle={60th IEEE Conference on Decision and Control (CDC)},
```
